api/passthrough/v1/passthrough.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package clutch.passthrough.v1;
```

Comment:
should we call it proxy instead of passthrough? just thinking of debugging down the line.

Reply:
True proxy makes more sense